src/views/ApartmentApp/components/StudentApplication/index.js

```diff
@@ -439,7 +444,7 @@ const StudentApplication = ({ userProfile, authentication }) => {
       result = false;
     }
     console.log('result of saving: ' + result); //! DEBUG
-    if (result !== null && result !== false && result !== -1 && typeof result === 'number') {
+    if (result && typeof result === 'number') {
```

Comment:
I assume you've changed the logic in the backend to no longer return `-1` for a bad result, since `-1` is a truthy value in JavaScript?

Reply:
Correct, we phased out the -1 a while ago, but it was left it for a bit until all the various features had a chance to be reviewed and merged